### ISSUE TITLE
Add WMOCAT to supported vocabulary

### ISSUE THREAD

**Issue by ferrighi** [metno/mmd#197]
To support the WMO Core profile, a new vocabulary should be added to the list in the documentation and thesauri. It is already included in the xslt although not used in the metadata records.

From WMO Core Profile specification:  
Requirement 8.2.1: Each WIS discovery metadata record shall include at least one keyword from the WMO_CategoryCode code list.

This controlled list is rather generic (see table 16 at https://wis.wmo.int/2012/metadata/WMO_Core_Metadata_Profile_v1.3_Part_2.pdf): https://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_CategoryCode 

Code: WMOCAT
Vocabulary: WMO_CategoryCode
Resource: https://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_CategoryCode

**Comment by steingod** [comment]
Yes, it would be more important to get OGC API Records according to WMO profile up and running.

**Comment by ferrighi** [comment]
Closing as outdated.
